app/views/settings/sharing/edit.html.haml

```diff
@@ -0,0 +1,20 @@
+= bootstrap_form_for(current_user, url: settings_sharing_path, method: :patch, data: { turbo: false }) do |f|
+  .card
+    .card-body
+      = f.form_group :sharing, help: t("activerecord.help.user.sharing_enabled") do
+        = f.check_box :sharing_enabled
+      = f.form_group :sharing, help: t("activerecord.help.user.sharing_autoclose"), class: false do
+        = f.check_box :sharing_autoclose
+
+  .card
+    .card-body
+      %h3= t(".advanced.title")
+      = t(".advanced.body_html")
+      = f.url_field :sharing_custom_url
```

Comment:
this field should probably have some validation in the model to ensure that `URI(...)` works and is HTTP(S), otherwise it'll break if you enter `asdfkjsaldfj` in there

Reply:
Validation in the backend has been added. `url_field` will also do HTML-side validation of the form submission (in browsers where that is supported at least).